server/@types/forms/index.d.ts

```diff
@@ -11,17 +13,14 @@ declare module 'forms' {
 
   export interface FindRecipientByPrisonerNameForm {
     prisonerName?: string
-    contacts?: Array<Contact>
   }
 
   export interface CreateNewContactByPrisonNumberForm {
```

Comment:
I dont have an obvious answer to this, and I'm certainly not suggesting more refactoring as part of this, but I find the classnames of these forms (and their controllers) a bit misleading, especially now we've removed fields from them.
EG: The form class is called `CreateNewContactByPrisonNumberForm` .... the pertinent part being "By Prison Number" ... yet the form doesnt include a prison number field

As mentioned I dont have any suggestions at this point, but just thought I'd raise it

Reply:
Yes - I know what you mean. I'll have a think on that one...